spec/support/authentication_helper.rb

```diff
@@ -0,0 +1,5 @@
+module AuthenticationHelper
+  def sign_in_user(user)
+    session[:user_id] = user.id
+  end
+end
```

Comment:
Check this PR: https://github.com/mla-wtag/ml_Journey/pull/6/files for an easier implementation of the helper method.

Reply:
thank you i did